src/orquestra/sdk/_base/_api/_wf_run.py

```diff
@@ -382,23 +379,62 @@ def get_results(self, wait: bool = False) -> t.Sequence[t.Any]:
                 state,
             )
         try:
-            results = (
-                *(
-                    serde.deserialize(o)
-                    for o in self._runtime.get_workflow_run_outputs_non_blocking(
-                        self.run_id
-                    )
-                ),
-            )
+            return self._runtime.get_workflow_run_outputs_non_blocking(self.run_id)
         except WorkflowRunNotSucceeded:
             raise
 
+    def get_results(self, wait: bool = False) -> t.Sequence[t.Any]:
+        """
+        Retrieves workflow results, as returned by the workflow function.
+
+        A workflow function is expected to return task outputs
+        (ArtifactFutures) or constants (10, "hello", etc.). This method returns values
+        of these. The order is dictated by the return statement in the workflow
+        function, for example `return a, b, c` means this function returns (a, b, c).
+        See also:
+        https://refactored-disco-d576cb73.pages.github.io/docs/runtime/guides/workflow-syntax.html
```

Comment:
**nitpick**: inb4 I know it's been here before. This is an internal link. Let's use the public one:
```suggestion
        https://docs.orquestra.io/docs/core/sdk/guides/workflow-syntax.html/workflow-syntax.html
```